05/DB.test.js

```diff
@@ -1 +1,142 @@
-import DB from './DB';
\ No newline at end of file
+import DB from './DB';
+
+describe('DB', () => {
+    describe('insert method', () => {
+        it('insert {a: 1, b: 2} obj to _rows', async () => {
+            const db = new DB()
+
+            const result = await db.insert({ a: 1, b: 2 })
+
+            expect(result).toStrictEqual({ a: 1, b: 2, id: 1 })
+        })
+
+        it('show error when id is not a number', () => {
+            const db = new DB()
+            const promise = db.insert({ id: '4' })
+
+            return expect(promise).rejects.toMatch('ID can be only number!')
+        })
+
+        it('show error when id already exists', () => {
+            expect.assertions(1);
+
+            const db = new DB()
+            const promise = db.insert({ id: 4 })
+                .then(() => db.insert({ id: 4 }))
+
+            return promise.catch((err) => {
+                expect(err).toBe('ID can\'t be duplicated!')
+            })
+        })
+    })
+    describe('select method', () => {
+        it('select item when id=3 is passed', () => {
+            const db = new DB()
+
+            const promise = db.insert({ a: 1, b: 2, id: 3 })
+                .then(() => db.select(3))
+
+            return promise.then(result => {
+                expect(result).toStrictEqual({ a: 1, b: 2, id: 3 })
+            })
+        })
+
+        it('show error when select id=4 and rows is empty', () => {
+            const db = new DB()
```

Comment:
Tutaj też trzeba pamiętaj o `.assetions()` bo inaczej przy braku błędu test przejdzie, a nie o to nam chodzi.